core/src/main/java/org/jdbi/v3/core/extension/Extensions.java

```diff
@@ -24,19 +24,21 @@
  * instances.
  */
 public class Extensions implements JdbiConfig<Extensions> {
-    private final List<ExtensionFactory> factories = new CopyOnWriteArrayList<>();
+    private final List<ExtensionFactory> factories;
 
     /**
      * Create an empty {@link ExtensionFactory} configuration.
      */
-    public Extensions() {}
+    public Extensions() {
+        factories = new CopyOnWriteArrayList<>();
```

Comment:
Again, now this initializer will have to be repeated in any additional constructors added.

Reply:
see symmetric constructors argument. There isn't really any prospect of having more than a default and copy constructor to Config classes